frontend/src/metabase/dashboard/components/DashboardEmbedHeaderButton/DashboardEmbedHeaderButton.tsx

```diff
@@ -49,7 +49,7 @@ export const DashboardEmbedHeaderButton = forwardRef(
         withArrow
         arrowSize={10}
       >
-        <Box>
+        <Flex>
```

Comment:
Using `Box` gives a div with `display: block` which adds unnecessary height since it fills up the container. With `display:flex` it takes the shape of its children so there's no more issues on that front. We still need this container to allow Mantine's tooltip component to have a target, and so that tooltips actually show up when running the e2e and unit tests.